Gemfile.lock

```diff
@@ -180,7 +180,7 @@ GEM
     jwt (2.3.0)
     launchy (2.5.0)
       addressable (~> 2.7)
-    libv8-node (16.10.0.0)
+    libv8-node (16.10.0.0-x86_64-darwin)
```

Comment:
What's going on with this commit? Something feels like its gone wrong?

Reply:
Fixed it!

Reply:
@CristinaRO @james worth noting that this *line* itself is not an error; it's pretty helpful. If we lose the `-x76_64-darwin` line, `bundle install` has to invoke the compiler for `libv8-node` instead of picking a pre-compiled platform-specific version for development.

Consequently, my machine has been compiling it for the last half-hour after pulling `develop`. I can't tell if @james was referring to the `RED` commit that this was in somehow having found its way onto this branch after rebase (because Github has no good way of commenting on a commit without referring to some file content) and @CristinaRO misunderstood and fixed the line @james was forced to refer to!

Reply:
We fixed this by finding the commit where we lost this line 183 on `devise-authentication`, rebasing interactively and editing that commit and running `git checkout HEAD -- Gemfile.lock && bundle install`.

This was caused by `rm Gemfile.lock && bundle install` recreating the Gemfile.lock without platform information. Bundler itself suggests the solution if you try and `bundle install` while a `Gemfile.lock` contains conflicts – it indicates that you should check out the last commit's version of Gemfile.lock and `bundle install` to resolve the conflict.